.github/workflows/lint.yaml

```diff
@@ -64,3 +64,27 @@ jobs:
           fail-on-findings: true
           find-copies: false
           find-renames: false
+  check-readme:
```

Comment:
As talked about on the other PR, I don't think this should be a check in this repo